cfme/tests/services/test_rest_services.py

```diff
@@ -449,3 +455,73 @@ def test_edit_blueprints(self, rest_api, blueprints, from_detail):
         assert len(edited) == response_len
         for i in range(response_len):
             assert edited[i].ui_properties == new[i]['ui_properties']
+
+
+class TestOrchestrationTemplatesRESTAPI(object):
+    @pytest.mark.tier(3)
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.7')
+    def test_create_orchestration_templates(self, rest_api, orchestration_templates):
+        """Tests creation of orchestration templates.
+
+        Metadata:
+            test_flag: rest
+        """
+        assert len(orchestration_templates) > 0
+        record = rest_api.collections.orchestration_templates.get(id=orchestration_templates[0].id)
```

Comment:
@mshriver The elements are created the same way regardless of the number, so check that the first one is ok was enough in my opinion. Edit and delete tests are more complex and are using the same fixture for resources creation, so this is just kind of sanity test. But I'm happy to change that if you have different opinion.